src/frame/settings.rs

```diff
@@ -88,7 +88,7 @@ impl Settings {
 
     pub fn set_max_frame_size(&mut self, size: Option<u32>) {
         if let Some(val) = size {
-            assert!(DEFAULT_MAX_FRAME_SIZE <= val && val <= MAX_MAX_FRAME_SIZE);
+            assert!((DEFAULT_MAX_FRAME_SIZE..=MAX_MAX_FRAME_SIZE).contains(&val));
```

Comment:
Same here I feel like the readability of this is worse here

Reply:
It's this lint: https://rust-lang.github.io/rust-clippy/master/index.html#manual_range_contains

Honestly, I'm inclined to agree; writing those did require some mental gymnastics. And I got one wrong at first and didn't realize until I run hyper's test suite.